File: 1-Iniciante/Ex1002.java
import java.util.Scanner;

public class Ex1002 { //para ser aceito no URI, mude o nome da classe para Main
    public static void main(String args[]){
        //variaveis
        double pi = 3.14159;
        double raio, area;
        Scanner entrada = new Scanner(System.in);

        //entrada
        raio = entrada.nextDouble();

        //processamento
        area = pi * Math.pow(raio, 2);

        //saida
        System.out.printf("A=%.4f\n", area);
        entrada.close();
    }
}

File: 1-Iniciante/Ex1002.kt
import java.util.Scanner
import kotlin.math.pow

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val raio = scanner.nextDouble()

    val PI = 3.14159
    val area = PI * raio.pow(2.0)
    println("A=${String.format("%.4f", area)}")
}

